modules/cudn-status-conditions.adoc

```diff
@@ -0,0 +1,84 @@
+//module included in the following assembly:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="cudn-status-conditions_{context}"]
+= ClusterUserDefinedNetwork status condition types
+
+The following tables explain the status condition types returned for `ClusterUserDefinedNetwork` CRs when describing the resource. These conditions can be used to troubleshoot your deployment.
+
+//The following table is subject to change and will be updated accordingly.
+
+.NetworkCreated condition types
+[cols="2a,2a,3a,6a",options="header"]
+|===
+
+|Condition type
+|Status
+2+|Reason and Message
+
+.3+|`NetworkCreated`
+.3+| `True`
+2+|When `True`, the following reason and message is returned:
+h|Reason
+h|Message
+
+|`NetworkAttachmentDefinitionCreated`
+|'NetworkAttachmentDefinition has been created in following namespaces: [example-namespace-1, example-namespace-2, example-namespace-3]'`
+
+.9+|`NetworkCreated`
+.9+| `False`
+2+|When `False`, one of the following messages is returned:
+h|Reason
+h|Message
+
+|`SyncError`
+|`failed to generate NetworkAttachmentDefinition`
+
+|`SyncError`
+|`failed to update NetworkAttachmentDefinition`
+
+|`SyncError`
+|`primary network already exist in namespace "<namespace_name>": "<primary_network_name>"`
+
+|`SyncError`
+|`failed to create NetworkAttachmentDefinition: create NAD error`
+
+|`SyncError`
+|`foreign NetworkAttachmentDefinition with the desired name already exist`
+
+|`SyncError`
+|`failed to add finalizer to UserDefinedNetwork`
+
+|`NetworkAttachmentDefinitionDeleted`
+|`NetworkAttachmentDefinition is being deleted: [<namespace>/<nad_name>]`
+|===
+
+.NetworkAllocationSucceeded condition types
+[cols="2a,2a,3a,6a",options="header"]
+|===
+
+|Condition type
+|Status
+2+|Reason and Message
+
+.3+|`NetworkAllocationSucceeded`
```

Comment:
Hi @stevsmit . Would you be OK to write the bug for https://issues.redhat.com/browse/OCPBUGS-44619 as you made a feature-doc-related update?

Reply:
@dfitzmau We don't need to document this one. It was an issue that affected 4.18 clusters when upgrading from 4.17. Customers never saw the issue in 4.17, so it's kind of weird to say "There was an issue when upgrading to 4.18. . . It has been resolved in 4.18" lol. Jean also confirmed we probably don't need to document this one.